base_local_planner/src/trajectory_planner_ros.cpp

```diff
@@ -387,7 +387,7 @@ namespace base_local_planner {
 
     std::vector<geometry_msgs::PoseStamped> transformed_plan;
     //get the global plan in our frame
-    if (!transformGlobalPlan(*tf_, global_plan_, global_pose, *costmap_, global_frame_, transformed_plan)) {
+    if (!transformGlobalPlan(*tf_, global_plan_, global_pose, *costmap_, global_frame_, ros::Time::now(), transformed_plan)) {
```

Comment:
and here, now() is later than the stamp we just got on the global_pose, so again, it would seem that approach would be reasonable.